docs/documentation/tips-and-tricks.md

```diff
@@ -15,30 +15,54 @@ You can change the service name by editing the file '/app/config.js'.
 
 ## Show navigation in the header
 
-Remove the comments surrounding the unordered list with an ID of proposition links.
-
-    <nav id="proposition-menu">
-      <a href="/" id="proposition-name">Service name</a>
-      <!--
-      <ul id="proposition-links">
-        <li><a href="url-to-page-1" class="active">Navigation item #1</a></li>
-        <li><a href="url-to-page-2">Navigation item #2</a></li>
-      </ul>
-      -->
-    </nav>
+Import the header component macro place it in the `{% block header %}`and provide `navigation` items as shown below.
```

Comment:
does this work fix something or just move it to macros? If its just about macros can we skip this commit, as we're not ready to push macros in prototyping